### PR TITLE
Add tags back migrations

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

There was an outage where frontend was trying to access a `tags` column that had been renamed to `deprecated_tags`. This may affect self hosted users who might not have run this migration. 

## How did you test this code?

Ran migrations locally and saw that tags column reappears.

### CODE REVIEW

**Review by rcmarron — APPROVED** [review]
LGTM

**Comment by tiina303** [comment]
Self-hosted folks might see errors between running these two migrations as the code would be still running, so I propose we make the old migrations no-ops & new-ones add if not exists. Note: we can do that retroactively later, we should land this ASAP

**Comment by rcmarron** [comment]
Don't merge this until we clean up the columns that were already added in prod

File: ee/models/event_definition.py [hunk 1]
```diff
@@ -6,17 +6,20 @@
...
-        "posthog.User", null=True, on_delete=models.SET_NULL, blank=True, related_name="verifying_user"
+        "posthog.User", null=True, on_delete=models.SET_NULL, blank=True, related_name="verifying_user",
     )
 
     # Deprecated in favour of app-wide tagging model. See EnterpriseTaggedItem
     deprecated_tags: ArrayField = deprecate_field(
         ArrayField(models.CharField(max_length=32), null=True, blank=True, default=list), return_instead=[],
     )
+    tags: ArrayField = deprecate_field(
+        ArrayField(models.CharField(max_length=32), null=True, blank=True, default=list), return_instead=[],
```
Comment (timgl) [inline ee/models/event_definition.py:24]: I'm pretty sure this is a blocking migration that we want to avoid (cc @EDsCODE )

**Review by timgl — CHANGES_REQUESTED** [review]

**Comment by tiina303** [comment]
We decided that self-hosted folks will just get a bit of 500s on the app (like we did & they have been in the past too) and we're not going to update the previous migrations (because it's tricky as they are used in a follow-up migration already)

**Pull request merged**